Softeq.XToolkit.Permissions.Droid/PermissionsManager.cs

```diff
@@ -20,8 +22,14 @@ public class PermissionsManager : IPermissionsManager
             _permissionsService = permissionsService;
             _permissionsDialogService = permissionsDialogService;
             _internalSettings = CrossSettings.Current;
-        }
-
+        }
+        
+        private bool IsPermissionRequested
+        {
+            get => _internalSettings.GetValueOrDefault(_isPermissionRequestedKey, false);

```

Comment:
please avoid using _internalSettings.GetValueOrDefault every time. Use internally saved property because of performance.

Reply:
used only for request permissions when check value, not frequent using